src/dropdown.ts

```diff
@@ -1926,7 +1926,7 @@ export default class Dropdown extends LitElement implements FormControl {
         event.preventDefault();
 
         const firstOption = [...this.#optionElementsNotHiddenIncludingSelectAll]
-          .reverse()
+          .toReversed()
```

Comment:
`eslint-plugin-unicorn` [added](https://github.com/sindresorhus/eslint-plugin-unicorn/releases/tag/v60.0.0) [`no-array-reverse`](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/no-array-reverse.md) to the recommended rules. So this update was required.